Confus/Confus/Audio/PlayerAudioEmitter.cpp

```diff
@@ -0,0 +1,50 @@
+#include "PlayerAudioEmitter.h"
+
+namespace Confus
+{
+    namespace Audio
+    {
+        PlayerAudioEmitter::PlayerAudioEmitter(irr::scene::IAnimatedMeshSceneNode* a_AttachedPlayer) : m_AttachedPlayer(a_AttachedPlayer)
+        {
+            createAudioSources();
+        }
+
+        PlayerAudioEmitter::~PlayerAudioEmitter()
+        {
+            delete(m_AudioSourceFootstep1);
+            delete(m_AudioSourceFootstep2);
+            delete(m_AudioSourceFootstep3);
+            delete(m_AttachedPlayer);
```

Comment:
I wasn't the one reviewing this, but this is bad. Do **not **delete this, you are not the one creating this object